src/groups/meta_group_wrapper.cpp

```diff
@@ -396,7 +396,11 @@ Napi::Value MetaGroupWrapper::memberGetAllPendingRemovals(const Napi::CallbackIn
     return wrapResult(info, [&] {
         std::vector<session::config::groups::member> allMembersRemoved;
         for (auto& member : *this->meta_group->members) {
-            if (member.is_removed()) {
+            auto memberStatus = member.status();
+            if (memberStatus == session::config::groups::member::Status::removed_unknown ||
+                memberStatus == session::config::groups::member::Status::removed ||
```

Comment:
I hate that Status is the only thing capitalized 😂 Not your problem but just need to let someone know

Reply:
Status is the name of the class enum, hence capitalized. 
The others are namespaces or values in the enum